apps/mobile/src/screens/NftSelectorScreen/NftSelectorPickerScreen.tsx

```diff
@@ -47,30 +48,24 @@ const NETWORKS: {
   })),
 ];
 
+const screenHeaderText = {
+  ProfilePicture: 'Select a profile picture',
+  Post: 'Select item to post',
+  Onboarding: 'Select profile picture',
```

Comment:
do we want `a profile picture` or just `profile picture`? looking at L52

Reply:
Previously, we used a `a profile picture`. Updated to use `profile picture` to standardized for both screen